api/envoy/extensions/network/dns_resolver/apple/v3/appl_dns_resolver.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
```

Comment:
This file should be called `apple_dns_resolver.proto`

Reply:
ACK